homeassistant/components/habitica/button.py

```diff
@@ -208,11 +200,7 @@ class HabitipyButtonEntity(StrEnum):
     HabiticaButtonEntityDescription(
         key=HabitipyButtonEntity.HEAL,
         translation_key=HabitipyButtonEntity.HEAL,
-        press_fn=(
-            lambda coordinator: coordinator.api.user.class_.cast["heal"].post(
-                targetId=coordinator.config_entry.unique_id
-            )
-        ),
+        press_fn=(lambda coordinator: coordinator.api.user.class_.cast["heal"].post()),
```

Comment:
```suggestion
        press_fn=lambda coordinator: coordinator.api.user.class_.cast["heal"].post(),
```

Reply:
there should be a ruff rule or something for this 😩